### DISCUSSION TITLE
The onboarding from Hobby -> Paid is poorly designed;  much trouble

### DISCUSSION THREAD

**Discussion by zadamg** [vercel/community#4301] [Account Issue]
Here was my path, in case it helps some others::::::

I have a hobby plan.  I'm getting ready to launch a product, and during testing, I get the 504 error due to the timeout of 10s.  My api cannot call for more than 10s.  So, I go to buy a Vercel plan for $20, which I was planning on doing anyway.  I paid immediately after realizing I have to in order to realize the maxDuration increase to 300 in my vercel.json, so..

I buy the plan... 

... but I have to make a team?... Ok. Make team.  Now I have hobby plan zadamg and team Twilitype. 

I attempt to run "vercel" again in CLI with my vercel.json setting my maxDuration to 300s. Still getting errors that I need to be on a paid plan.  After digging through the documentation I do a "vercel switch" to my new team.

Great! I'm in the new scope for this new team, so I try "vercel" again...
```
❗️  Your Project was either deleted, transferred to a new Team, or you don’t have access to it anymore.
```
Ok... nothing telling me what I should do about it. Or something to say, "Hey, you just bought a plan.. do this!"  Ok, so maybe I need to perform a "transfer" a project from my zadamg account to my new "Tea".  So I do the transfer. Now, zadamg my Hobby has 2 old projects and my Twilitype has the main project I'm trying make work again.  So, I run "vercel" in the cli yet again... and get the same error message that my project has been deleted or transferred.  But I'm in that team's scope in the CLI and the project is in the Overview tab in my dashboard..

I go to Support, because now I can create a "Case", but when I try to select my project from the list, the only two selectable projects are the OLD projects on my HOBBY account but hobby accounts can't submit support tickets.  so what do?  The project I moved to my Team account that I just bought is not in the list for the Support Case form.... So I can't even submit a support ticket!  <--- this part might be a legitimate bug, honestly.

_________________
It could just be me, but I don't think I missed any steps on the on-boarding. 

Thanks in advance; I appreciate the server otherwise and have been pleased with it,

Zack

**Comment by amyegan** [comment]
I'm sorry you had such a rough Pro signup experience. Thank you for sharing this feedback! I'm sending it to the teams who handle these areas so we can improve our Pro onboarding flow. 🙇
